### PR TITLE
week3

### PR DESCRIPTION
**Pull request by sarross88**
Week 3 assignment

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,30 +1,51 @@
 import './App.css'
 import TodoList from './TodoList.jsx'
 import AddTodoForm from './AddTodoForm.jsx'
```
Comment (taiwo2) [inline src/App.jsx:3]: this is imported twice. so I can't run your code

File: src/AddTodoForm.jsx [hunk 2]
```diff
@@ -1,9 +1,24 @@
-const AddTodoForm = () => {
+import { useState } from 'react';
+
+const AddTodoForm = (props) => {
+
+const [toDoItem, setTodoItem] = React.useState('');
+
+console.log(`rendering todo with toDoItem: ${toDoItem}`);
+
+const handleTodoChange = (event) => {
```
Comment (taiwo2) [inline src/AddTodoForm.jsx:9]: kindly modify the function based on the project requirements

**Review by taiwo2 — COMMENTED** [review]
Your code is not running and has bugs to fix. please revisit the project and implement it as project per the instructions

File: src/App.jsx (hunk 1 above)
Comment (sarross88) [reply src/App.jsx:3]: Fixed!

**Pull request merged**